src/d3a/d3a_core/simulation.py

```diff
@@ -326,18 +325,24 @@ def _execute_simulation(self, slot_resume, tick_resume, console=None):
 
             sleep(seconds_until_next_tick)
 
-        simulation_time_counter = time.time()
+        if self.slot_length_realtime:
+            tick_length_realtime_s = self.slot_length_realtime.seconds / \
+                                     self.simulation_config.ticks_per_slot
+            # flag for reporting whether slowdown needed to be applied (True)
+            # or if the actual runtime is higher than the set slot_length_realtime (False):
+            slow_down_applied = True
+
         for slot_no in range(slot_resume, slot_count):
             self._update_progress_info(slot_no, slot_count)
 
-            log.warning(
-                "Slot %d of %d (%2.0f%%) - %s elapsed, ETA: %s",
-                slot_no + 1,
-                slot_count,
-                self.progress_info.percentage_completed,
-                self.progress_info.elapsed_time,
-                self.progress_info.eta
-            )
+            slot_log_str = f"Slot {slot_no + 1} of {slot_count} - " \
+                           f"({round(self.progress_info.percentage_completed, 1)}%) " \
+                           f"{self.progress_info.elapsed_time} elapsed, " \
+                           f"ETA: {self.progress_info.eta}"
+            if self.slot_length_realtime:
+                slot_log_str += f" (slowdown applied: {slow_down_applied})"
```

Comment:
I am not sure that just one boolean flag, which will be the same for all slots and printed for all of them, is that helpful for debugging. IMO it is not needed here